driver-core/src/test/java/com/datastax/driver/core/CCMBridge.java

```diff
@@ -144,7 +148,7 @@
     private static final String CCM_COMMAND;
 
     static {
-        CASSANDRA_VERSION = System.getProperty("cassandra.version");
+        INPUT_CASSANDRA_VERSION = System.getProperty("cassandra.version");
```

Comment:
What about distinguishing two different system properties for default versions to use: one for OSS C*, one for DSE? 

One could e.g. specify that tests requiring DSE should by default use 5.0.3, but tests requiring OSS C* should rather use 3.10.

In this case, the `dse` system property is useless and the mapping from a DSE version to an OSS C* one should only be used when determining the corresponding C* version of a DSE version (which might be different than the "global" C* version).

I can give it a try myself if you don't mind.